.jenkins/Jenkinsfile

```diff
@@ -86,14 +86,10 @@ def checkPreCommitRequirements() {
                 timeout(15) {
                     // This is run to test that it works with the dependencies
```

Comment:
Lines 75 and later i.e. this section still references simulation. Please fix this.
 stage('Sim default compiler') {
        // This particular test asserts that everything (at least
        // for simulation) can be built after using our
        // install-prereqs ansible playbook to to bootstrap a machine.
        node {